just_audio_platform_interface/lib/just_audio_platform_interface.dart

```diff
@@ -1025,52 +1027,58 @@ class ProgressiveAudioSourceMessage extends UriAudioSourceMessage {
     required String uri,
     Map<String, String>? headers,
     dynamic tag,
-  }) : super(id: id, uri: uri, headers: headers, tag: tag);
+    Map<String, String>? drm,
+  }) : super(id: id, uri: uri, headers: headers, tag: tag, drm: drm);
 
   @override
   Map<dynamic, dynamic> toMap() => <dynamic, dynamic>{
         'type': 'progressive',
         'id': id,
         'uri': uri,
         'headers': headers,
+        'drm': drm,
       };
 }
 
 /// Information about a DASH audio source to be communicated with the platform
 /// implementation.
 class DashAudioSourceMessage extends UriAudioSourceMessage {
-  DashAudioSourceMessage({
-    required String id,
-    required String uri,
-    Map<String, String>? headers,
-    dynamic tag,
-  }) : super(id: id, uri: uri, headers: headers, tag: tag);
+  DashAudioSourceMessage(
+      {required String id,
+      required String uri,
+      Map<String, String>? headers,
+      dynamic tag,
+      Map<String, String>? drm,})
```

Comment:
There is a trailing comma here, but the formatter hasn't been run yet, so these 6 lines are showing up as changes.

(I'll stop here, but please view your PR in a side-by-side diff to see which lines you have actually changed. If you have made a change due to formatting, you can try to revert those formatting changes back to the way they were originally.)

Reply:
I have checked the formatting. The methods have gone to multi line formatting as the DRM property was added and guess exceeded something on formatter. Should I manually format to add the property to remain on one line?(if removed and format then it goes to one line)

Reply:
I guess it is a problem with your IDE, but from the command-line you could run the dart format command (either `dartfmt` or `dart format`)

Reply:
When I run the formatter it correctly formats it like this:

```dart
  DashAudioSourceMessage({
    required String id,
    required String uri,
    Map<String, String>? headers,
    dynamic tag,
    Map<String, String>? drm,
  }) : super(id: id, uri: uri, headers: headers, tag: tag, drm: drm);
```

Reply:
Thanks for the advice. I have done as asked and committed the formatted file. (Hopefully better now and as expected)

Reply:
Done in all places.